source/reference/minio-mc.rst

```diff
@@ -418,6 +418,34 @@ The following list describes each possible file path location in the order
 
 You can use the ``--config-dir``
 
+.. _minio-mc-certificates:
+
+Certificates
+------------
+
+The MinIO Client stores certificates and CAs for deployments with :ref:`server side encryption <minio-sse>` to the following paths:
```

Comment:
I don't *think* sse matters here.

The `mc` client has to have a CA for the MinIO deployment it's connecting to - so that just requires the MinIO Server to enforce TLS.

If the MinIO Server TLS cert were self signed, you would need to put the public key into the mc CA directory (or something like that) to connect without `--insecure`.

For SSE I don't _think_ mc directly interfaces with KES. It would do so through MinIO's internal kes-go client, so the CA there wont matter as much.

Reply:
Cut the SSE bit.